nss-day-cohort-45/nutyshell-running-with-scissors-1
# Nd taskcss

# Description
new CSS for tasks and tasks dialog
fixed breaking of delete/completion of tasks produced by CSS
fixed overflow issue with task list

Fixes # (issue)
## Type of change
Please delete options that are not relevant.


# Testing Instructions
-check dialog for standard CSS used for all dialogs
-check deleting a task still works
-check completing of task with checkbox hides task
-check overflow scroll on task list

# Checklist:
- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] My changes generate no new warnings or errors
